### ISSUE TITLE
SelectionManager stops filtering externally if JSON filter is applied

### ISSUE THREAD

**Issue by darthtrevino** [Microsoft/powerbi-models#53]
Hello,
I'm trying to update a visual so that it can filter across two dimensions (x-axis = time, y-axis = categories). The base behavior was that it was using the selectionManager to select categories. I've been experimenting with using `applyJsonFilter` to filter across the time dimension as well. That part is working.

What's broken is that the selectionManager no longer pushes selections out to other visuals. It happens the moment the "filter" property enters capabilities.json. The selectionManager appears to be selecting items internally just fine.

Is this a known limitation for custom visuals?

**Comment by ali-hamud** [comment]
Hi @darthtrevino,

I am moving your question to the custom visuals team.
have you managed to solve your problem?

**Comment by darthtrevino** [comment]
No, we've had to reduce the intended functionality of one of our visuals
due to this

**Comment by lasalame** [comment]
if “filter” attribute is defined in capabilities files, the Selection manager won’t work until you call applySelectionFilter method 
so selection and applyJson filter both use “filter” attribute
